script/Ciel/Day 12/12_01_CIEL12_1_A.txt

```diff
@@ -12,7 +12,7 @@ Just as I was thinking that today's knocks were a bit louder than usual,
 [sha:27beaea45ec2140eebfbc52e09898f6c2782045c]{
 -- Page 6, Offset 30880.
 -- 「おはようございます志貴さん。今朝の具合はどうですか？」
-"Good morning Shiki-san. How are you feeling this morning?"
+"Good morning, Shiki-san. How are you feeling this morning?"
```

Comment:
Don't love double-morning, maybe change second to `today`

Reply:
nice catch